README.md

```diff
@@ -49,6 +52,31 @@ be sourced in your `.bashrc` or `.cshrc` which will make `xtb` and the
 parameter files available. Also a pkg-config file is generated to make
 `xtb` available in other projects as dependency.
 
+### CMake
+
+The CMake build system requires both `make` and `cmake` to be installed, the
+latter has to be version 3.9 or newer.
+
+Building XTB with CMake works with the following chain of commands:
+
+```bash
+mkdir build
+cd build
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/some/place/nice ..
+make
+make install
+```
+
+The compiled tests can be executed using the following commands:
+
+```bash
+export XTBHOME=$PWD
```

Comment:
XTBHOME -> XTBPATH
also I don't want to encourage wildly exporting variables. Better set it locally by
```
XTBPATH=$PWD make test
```